slajkoa_hf_v2.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="hu">
+<head>
+    <meta charset="UTF-8" />
+    <title>Házi_slajkoa</title>
+</head>
+
+<style>

```

Comment:
Érdemes bedobni az elkészült HTML-t a validátorba, egy-két könnyen javítható hibát ki tud szúrni: https://validator.w3.org/nu/#textarea